test/rules/variable-name/all-caps-for-const/test.ts.lint

```diff
@@ -0,0 +1,34 @@
+var snake_case_var;
+    ~~~~~~~~~~~~~~  [regular-case]
+
+var camelCaseVar;
+
+var PascalCaseVar;
+    ~~~~~~~~~~~~~  [regular-case]
+
+var UPPER_CASE_VAR;
+    ~~~~~~~~~~~~~~  [err]
+
+let snake_case_let;
+    ~~~~~~~~~~~~~~  [regular-case]
+
+let camelCaseLet;
+
+let PascalCaseLet;
+    ~~~~~~~~~~~~~  [regular-case]
+
+let UPPER_CASE_LET;
+    ~~~~~~~~~~~~~~  [err]
+
+const snake_case_const;
+      ~~~~~~~~~~~~~~~~  [regular-case]
+
+const camelCaseConst;
+
+const PascalCaseConst;
+      ~~~~~~~~~~~~~~~  [regular-case]
+
```

Comment:
could we add a test case for something like this?

```
class Test {
    public static readonly MY_FIELD = 10;
}
```

I think the above code shouldn't trigger a rule violation